### ISSUE TITLE
Property is Theft shootout cheatin' interaction not working properly

### ISSUE THREAD

**Issue by Caelreth1** [townteki/townsquare#1620]
**Card**
Property Is Theft

**Describe the bug**
When the opponent pulls a cheatin' hand during a shootout and I do not, I lose 1 ghost rock, or if I have 0 ghost rock, I do not gain any ghost rock. ( I have only tested this against automatons)

**To Reproduce**
1. Choose Property is Theft as your base
2. Start a shootout
3. Wait for the opponent to reveal a cheatin' hand

**Screenshots**
See enclosed screenshot, note that my ghost rock total is still 0, despite the log reporting that my opponent has paid me 1.
![PropertyIsTheft](https://github.com/townteki/townsquare/assets/151785921/6419188a-7065-417f-af5b-5cb1f6b2bf9a)
**Additional context**

**Comment by mmeldo** [comment]
This one is funny. This is how the code looks like:
```javascript
                        this.game.transferGhostRock({
                            from: context.player,
                            to: opponent,
                            amount: 1
                        });
```
from which you can see it is other way around 😄 Looks like no one is playing with that outfit since no one noticed.

**Comment by mmeldo** [comment]
Should be fixed by `1.1.7`
